src/button-icons/buttons-icons.css

```diff
@@ -0,0 +1,60 @@
+.omrs-btn-icon-medium {
+  width: 24px;
+  height: 24px;
+  text-align: center !important;
+  background: transparent;
+  border: none !important;
```

Comment:
using `!important` might not allow overriding this property on the client. @joeldenning what are your thoughts on this?